src/download_data.py

```diff
@@ -0,0 +1,40 @@
+# author: Son Chau
+# date: 2021-11-19
+
+"""This script prints out docopt args.
+Usage: download_data.py --url=<arg1> --path=<arg2>
+
+Options:
+--url=<arg1>             URL from where to download the data (must be zip file)
+--path=<arg2>            Path (not including filename) of where to locally write the file
+
+Example:
+python download_data.py --url=http://www3.dsi.uminho.pt/pcortez/wine/winequality.zip --path=./data
+"""
+
+from docopt import docopt
+from urllib.request import urlopen
+from io import BytesIO
+from zipfile import ZipFile
+
+# define main function
+def main():
+
+    # parse arguments
+    args = docopt(__doc__)
+
+    # assign args to variables
```

Comment:
I would also remove this and keep these two lines under the "parse arguments" comment.